.github/workflows/maven.yml

```diff
@@ -0,0 +1,23 @@
+name: Java CI
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+        java: [8, 8.0.192, 11, 11.0.2, 13, 13.0.4, 15, 16-ea]
+      fail-fast: false
+      max-parallel: 2          
+    name: Test JDK ${{ matrix.java }}, ${{ matrix.os }}
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up JDK
+      uses: actions/setup-java@v1
+      with:
+        java-version: ${{ matrix.java }}
+    - name: Test with Maven
+      run: mvn test -B --file pom.xml
```

Comment:
@geertjanw I think the command here should be the same as in other cases: ``mvn clean install -Pcheckstyle,itcases``

Can you modify it please?

Reply:
But then we're building and not testing, i.e., not running tests only, which is what this GitHub Action is about. We can change it to do that, but that's something different.

Reply:
@geertjanw if ``mvn test`` runs both unit tests (surefire plugin) and integration tests (failsafe plugin), then I am ok with it. 

If not, please specify the full command : D

Reply:
Yup, it does. :-)